moment-js.html

```diff
@@ -1,5 +1,6 @@
-<link rel="import" href="../polymer/polymer.html">
-<link rel="import" href="./moment-import-behavior.html">
+<link rel="import" href="../polymer/polymer-element.html">
+<!--<link rel="import" href="./moment-import-behavior.html">-->
+<script src="../moment/min/moment.min.js"></script>
```

Comment:
We were using a behavior to ensure not loading `moment.min.js` each time a `moment-js` tag is attached to the DOM. Have you tested it before remove it?